plugins/app/src/components/module/Module.tsx

```diff
@@ -13,6 +13,12 @@ import { useDynamicScript } from '../../hooks/useDynamicScript';
 //   return;
 // };
 
+/**
+ * Load and initialize a federated module via webpack container.
+ * @param scope the module scope. In kobs it's usually the plugin name: e.g. 'kiali'
+ * @param module the name of the module entry point. e.g. './Page'
+ * @returns the module component or method
+ */
```

Comment:
Added some documentation. In a later PR, I'll try to type this with generics so that we can get rid of `any`.